types/wordpress__sync/index.d.ts

```diff
@@ -19,56 +19,22 @@ declare module '@wordpress/sync' {
 	type ProviderCreator = (
 		objectType: ObjectType,
 		objectId: ObjectID,
-		ydoc: Y.Doc,
-		awareness?: Awareness
+		ydoc: Y.Doc
 	) => Promise< ProviderCreatorResult >;
 
 	// Only include what we actually use from SyncConfig.
 	interface SyncConfig {
```

Comment:
just remove this interface, we're not using it anymore

Reply:
Done in https://github.com/Automattic/vip-real-time-collaboration/pull/72/commits/420b3c53f5bf51d1080d91ffce3e0023b465fe41